stable/postgresql/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgresql
-version: 3.18.4
+version: 3.18.6
```

Comment:
```suggestion
version: 3.18.5
```

Reply:
I've opened another PR where I'm bumping-up the version to 5.